src/labeler.ts

```diff
@@ -216,7 +216,7 @@ function checkAll(changedFiles: string[], globs: string[]): boolean {
 
 function checkBranch(glob: string): boolean {
   const matcher = new Minimatch(glob);
-  const branchName = github.context.ref;
+  const branchName = github.context.payload.pull_request!.head.ref;
   core.debug(` checking "branch" pattern against ${branchName}`);
   core.debug(`  - ${printPattern(matcher)}`);
   if (!matcher.match(branchName)) {
```

Comment:
Because the types of the `WebhookPayload` are `[key:string]: any` might be best to avoid the type assertion and just add an extra check
```suggestion
  const branchName = github.context.payload.pull_request?.head.ref;
  core.debug(` checking "branch" pattern against ${branchName}`);
  core.debug(`  - ${printPattern(matcher)}`);
  if (branchName || !matcher.match(branchName)) {
```

Reply:
I thought about that, but figured it would not be necessary because it [shouldn't be possible](https://github.com/joshdales/labeler/blob/79c0cc7c0bec2419725a656f8314f11c21899821/src/labeler.ts#L24) to end up here [without a `pull_request`](https://github.com/joshdales/labeler/blob/79c0cc7c0bec2419725a656f8314f11c21899821/src/labeler.ts#L68).

I'm happy to make the change though.

Reply:
Thanks! Yeah even though it should be there github just provided us with some bad type that make me nervous 😆